src/com/google/common/css/compiler/gssfunctions/GssFunctions.java

```diff
@@ -370,7 +375,7 @@ public String getCallResultString(List<String> args)
           baseColorString, args.get(1), args.get(2), args.get(3));
     }
 
-    protected String addHsbToCssColor(
+    public static String addHsbToCssColor(
```

Comment:
I'm working on it. I'm adding some methods to manipulate colors using HSL color space (which make more easy some operations).
Yes I can extract some functions and inherit from a base class, it'll be better.